time_tracker_api/time_entries/time_entries_dao.py

```diff
@@ -93,21 +94,35 @@ def build_custom_query(self, is_admin: bool, conditions: dict = None):
     def get_all(self, conditions: dict = None, **kwargs) -> list:
         event_ctx = self.create_event_context("read-many")
         conditions.update({"owner_id": event_ctx.user_id})
-
+        is_complete_query = conditions.get("user_id") == '*'
```

Comment:
I'll move this variable next to its usage, before line 116

Reply:
Sure, that was my first option, but this variable must be declared here because after it goes through `custom_query` method and these values ​​are lost.